src/mesh/mesh.hpp

```diff
@@ -130,6 +131,8 @@ class Mesh {
   int step_since_lb;
   int gflag;
 
+  bool do_coalesced_comms;
```

Comment:
should this be `const`? Not super important though

Reply:
yeah, I think it could be `const`

Reply:
done